saleor/product/tasks.py

```diff
@@ -183,3 +184,15 @@ def update_products_search_vector_task():
         :PRODUCTS_BATCH_SIZE
     ]
     update_products_search_vector(products, use_batches=False)
+
+
+@app.task(queue=settings.COLLECTION_PRODUCT_UPDATED_QUEUE_NAME)
+def collection_run_product_updated_task(product_ids):
+    manager = get_plugins_manager()
+    products = list(
+        Product.objects.filter(id__in=product_ids).prefetched_for_webhook(
+            single_object=False
+        )
+    )
+    for product in products:
+        manager.product_updated(product)
```

Comment:
You can get webhook
`webhooks = get_webhooks_for_event(WebhookEventAsyncType.PRODUCT_UPDATED)`
and pass it to `manager.product_updated(product, webhooks=webhooks)` so you wont be fetching webhook for every product.